ddpui/api/transform_api.py

```diff
@@ -576,15 +576,15 @@ def delete_operation(request, operation_uuid):
 
 
 @transformapi.get('/dbt_project/data_type/', auth=auth.CanManagePipelines())
-def get_postgres_datatypes(request):
+def get_warehouse_datatypes(request):
+    """Get the datatypes of a table in a warehouse"""
     orguser: OrgUser = request.orguser
     org = orguser.org
 
     org_warehouse = OrgWarehouse.objects.filter(org=org).first()
     if not org_warehouse:
         raise HttpError(404, "please setup your warehouse first")
     
-    wclient = _get_wclient(org_warehouse)
-    data_types = wclient.get_column_data_types()
+    data_types = dbtautomation_service.warehouse_datatypes(org_warehouse)
     return Response(data_types)
```

Comment:
we usually return without wrapping in `Response`... i don't know what this does, whether it makes a difference to the response, or whether it is good style... all i know is that we don't explicitly wrap our responses in `Response` objects